apps/students/models.py

```diff
@@ -7,6 +7,9 @@
 from home.models import Calendar, Schedule
 
 
+from django.urls import reverse
```

Comment:
This import statement should be put with the django imports at the top in alphabetical order.

Reply:
done sir..now it's in alphabetical order